DataFormats/FWLite/examples/patZpeak.py

```diff
@@ -39,7 +41,7 @@
     muons = handle.product()
     # use muons to make Z peak
     numMuons = len (muons)
-    if muons < 2: continue
+    if numMuons < 2.0: continue
```

Comment:
Could you still change this to
```suggestion
    if numMuons < 2: continue
```
? Could you also squash the commits into one so that the jpsi file doesn't appear in the history? Thanks!